### ISSUE TITLE
djackddonovan pull requests on the master?

### ISSUE THREAD

**Issue by Darthagnon** [haganbmj/MagicSetEditor2#55]
I noticed @djackddonovan opened a few pull requests on the MSE master (his fork: https://github.com/djackddonovan/MagicSetEditor2). Was wondering if they could be integrated here, since this seems to be where development is happening?

**Comment by haganbmj** [comment]
Yeah, I haven't been the most active, but if there's people interested in making changes that'd be great.

**Comment by Darthagnon** [comment]
![chrome_240615_104](https://github.com/haganbmj/MagicSetEditor2/assets/20792748/a79accc6-32c5-4c4f-83e0-43a093f4f92d)

**Comment by haganbmj** [comment]
Huh, I would've guessed it's because you're not a collaborator on their fork, but it looks like I was able to create one https://github.com/haganbmj/MagicSetEditor2/pull/77 so maybe permissions when you don't have access to either fork are just really messy.

I expect the PR I opened is going to be a struggle too since I don't have permission to commit back changes to correct the merge conflicts. Might have to copy that branch into a repo I do have permissions on to make that feasible.

**Comment by haganbmj** [comment]
To make it more confusing maybe I think I had cherrypicked some of those changes way back when.
Here's at least one example.

https://github.com/haganbmj/MagicSetEditor2/pull/53

**Comment by Darthagnon** [comment]
https://github.com/haganbmj/MagicSetEditor2/pull/77
